waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -130,3 +130,34 @@ proc getMerkleRoot*(rlnInstance: RLN[Bn256]): MerkleNodeResult =
   var rootValue = cast[ptr array[32,byte]] (root.`ptr`)
   let merkleNode = rootValue[]
   return ok(merkleNode)
+
+proc toMembershipKeyPairs*(groupKeys: seq[string]): seq[MembershipKeyPair] {.raises: [Defect, ValueError]} =
+  ## groupKeys is an alternating sequence of identity keys and their corresponding id commitment keys in the hexadecimal format
```

Comment:
Could this be a list of tuples instead? This seems like a pretty ad hoc data structure that will be prone to bugs

Reply:
Done!